chef/cookbooks/hyperv/recipes/config_nova.rb

```diff
@@ -52,10 +52,14 @@
   end
 end
 
+directory node[:openstack][:nova][:config] do
+  action :create
+end
+
 # Chef 11.4 fails to notify if the path separator is windows like,
 # according to https://tickets.opscode.com/browse/CHEF-4082 using gsub
 # to replace the windows path separator to linux one
-template "#{node[:openstack][:config].gsub(/\\/, "/")}/nova.conf" do
+template "#{node[:openstack][:nova][:config].gsub(/\\/, "/")}/nova.conf" do
```

Comment:
Performance/StringReplacement: Use `tr` instead of `gsub`. (https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)